hawk/app/controllers/api/v1/status_controller.rb

```diff
@@ -2,7 +2,8 @@ module Api
   module V1
     class StatusController < ApiController
       before_action :get_status
-
+      attr_reader :current_user
+      
```

Comment:
The error occurs when @current_user is nil in this scope. By passing @current_user from the parent class to the child it works. I'm actually wondering why it worked most of the time. Maybe because the garbage collector hasn't done its job at this point :D